### ISSUE TITLE
Evaluate/update ClamAV to upstream head

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#4370]
### Tasks

```[tasklist]
- [x] Branch our main
- [x] Pull from upstream
- [x] Merge
- [x] Evaluate
```

**Comment by asteel-gsa** [comment]
PR Available for review: https://github.com/GSA-TTS/clamav-rest/pull/13

**Comment by asteel-gsa** [comment]
> PR Available for review: [GSA-TTS/clamav-rest#13](https://github.com/GSA-TTS/clamav-rest/pull/13)

Merged.
#2 https://github.com/GSA-TTS/clamav-rest/pull/14
